package.json

```diff
@@ -33,10 +33,10 @@
   "dependencies": {
     "@reach/accordion": "^0.16.1",
     "@reach/tabs": "^0.16.4",
-    "@svgr/webpack": "^5.5.0",
+    "@svgr/webpack": "^6.0.0",
```

Comment:
major bump so good to review: The changelog does not show it though.
There is a github release for [v6](https://github.com/gregberge/svgr/releases/tag/v6.0.0).

> In most cases you will have nothing to change.

That's good.

From the [migration guide](https://react-svgr.com/docs/migrate/):
> @svgr/webpack is now only compatible with webpack 5.

We do use webpack v5 ✅

> SVGR v6 now uses SVGO v2 and does not merge config any more.
> If you have a custom SVGO configuration (using a dedicated file or using svgo key in SVGR config), then please follow SVGO migration guide.

Oh oh, I don't know what `gatsby-plugin-svgr` uses under the hood the [sources](https://github.com/zabute/gatsby-plugin-svgr/blob/5178de44d0cdda334c030ce0b19cf095bdbb2b9d/gatsby-node.js) don't show any configs regarding svgo.

We don't have any local config of svgo for storbyook (see the webpack adjustment in `.storybook/main.js`)